.github/workflows/ci-for-default-ns.yaml

```diff
@@ -75,15 +75,24 @@ jobs:
           # for testing start-debug and stop-debug deployment
           # mons
           kubectl rook_ceph debug start rook-ceph-mon-a
-          sleep 5
+          sleep 10
```

Comment:
```suggestion
```
why use sleep again, the goal of the pr is to remove the sleep?

Reply:
These timeout's are there for certain reasons,

Like Subham this one is for whenever we run the  ` kubectl rook_ceph debug start rook-ceph-mon-a`
we have to wait till the debug pod should triggered and in initialising phase as sometimes we just trigger the pod to create and if api response is not quick we won't get any pod in that namespace that is been starting

Reply:
So if there is no pod created yet, the script fails to wait for the condition? Could we change the script to handle the case where the pod doesn't exist?

Reply:
Thanks, Addressed them :)

Reply:
I was thinking about this a bit differently... As part of the loop inside `wait_for_condition`, it should just wait for the pod to exist before also checking that the pod is running. Then we don't need to have the sleep and check for a pod duplicated everywhere.

Reply:
Okay addressed it